packages/components/src/components/Dialog/DialogHeader.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { EDSStyleSheet } from "../../styling";
+import { TextChildren, Typography } from "../Typography";
+import { View } from "react-native";
+import { useStyles } from "../../hooks/useStyles";
+
+export const DialogHeader = (props: TextChildren) => {
+    const styles = useStyles(themeStyles);
+    return <View style={styles.header}>
+        <Typography variant="h6" style={styles.title} >{props.children}</Typography>
+    </View>
+
+}
+
+const themeStyles = EDSStyleSheet.create(theme => ({
```

Comment:
magic values should reference the master token instead

Reply:
Fixed